site-setup/learn_article.ftl

```diff
@@ -58,26 +58,25 @@ html{box-sizing:border-box;-ms-overflow-style:scrollbar}*,::after,::before{box-s
 					<label for="productDocumentationSelector">
 						<select class="form-control" id="productDocumentationSelector">
 							<option value="analytics-cloud">
-								{% trans %}Analytics Cloud{% endtrans %}
```

Comment:
Can you please update all the places with trans to use the languageUtil like `${languageUtil.get(locale, "analytics-cloud", "Analytics Cloud")}`?